### ISSUE TITLE
1.6.0 Feedback and Issues

### ISSUE THREAD

**Issue by TechnoMysterio** [daedalus4096/ThaumicWonders#38]
I'm back with my usual endless list:

- Catalysation Chamber now works properly from my testing. Good job. I can also confirm that the Ore Diviner issue is resolved.
- Putting 'Temple' into the Structure Finder tends to lead me to Witch Huts. Is this supposed to happen? Might this also happen with something like Igloos?
- Suggestion for Structure Finder: perhaps allow it to have multiple pages so that more specific structures can be located, so Jungle and Desert Temples instead of just the generic 'Temple'. It might even be worth adding Thaumcraft Structures like the Hollow Hill or Infested Tree, or ones from other mods like Quark Pirate Ships.
- Meaty Orb is... quite something. I suddenly fancy a barbeque.
- Visual FX tweaks working well, both the Starfield on the Ripper and the Void Portal Instability wobbling.
- Question: Is there a limit to how big can a Ripper grow a Rift to? The book entry mentions diminishing returns, but is there a particular size limit that can be reached?
- Suggestion: colourable Flying Carpets. I thought I might have seen this mentioned elsewhere, but colourable carpets would be good, in the standard 16 base colours.
- It might be worth updating Wonders' CurseForge page. It doesn't really reflect all the new content that has been added to it, and it might be better to have one showcase video instead of multiple ones for different things.

Good work as always. As a brief question: got anything interesting planned? I'm always curious to see what people might make for addons.

**Comment by daedalus4096** [comment]
- Thanks for letting me know that the ore dict issues are fixed.  It was fine on my end, but independent confirmation is always a relief.
- Unfortunately, it's "intended" behavior that "Temple" leads to not just Desert and Jungle Temples, but also Witch Huts and Igloos.  It's a limitation of the core Minecraft API.  MC 1.13 expands the selection of options to differentiate between those, but we don't have 1.13.  
- As for adding non-vanilla structures, I'd have to do a serious deep dive into the core Minecraft code to figure out how to do that.  There's an API call in World that returns the stuff I currently allow searching for.  The code it calls is big and scary.
- I'm glad you enjoy the Meaty Orb.  Implementing it was an enjoyable exercise in silliness and puns.
- I don't enforce an upper limit on rift size and to my knowledge neither does TC.  The size of the rift is proportional to the square root of the essentia used to fuel the reaction, and the drop chances of the rift are based on the square root of its size.  So you can pour a *lot* of essentia down that hole before hitting the functional bottom.  Who am I to say someone can't do that?  That said, if you're not careful, the rift may end up eating one of your Rippers, and then you'll be sad.
- Yeah, I've seen the suggestion for dyeable Flying Carpets.  I might do it, but it's not high on my list of priorities.  I really like what my artist friend came up with for the base color scheme.
- As for what's next, I'm not too sure.  I've already done everything I initially set out to do and then some.  There are some smaller things on my to-do list, like a couple of bauble ideas.  Two things I'd definitely like to do are Void Conjuration (creating blocks from essentia and rift-stuff) and Directed Enchantment (an enchantment mechanic that lets you choose what you want instead of being random), but I'm expecting those are both going to be bigger projects.

**Comment by TechnoMysterio** [comment]
- It'd make sense to at least support Thaumcraft structures, even though there's only a few in TC6.
- Well I did play about with Rifts with NBTEdit, and once I set a Rift's size to 400 or 1000 or something stupid and it grew insanely large, so I can confirm there seems to be no upper limit.
- Regarding Rift content and expansions, me and TheCodex6824 have been planning things around that for Augmentation, and we've already run into problems where we wanted to have Rift creation tools, but since Wonders has the Ripper we need to rework our plans so that the content isn't too redundant. Thinking forward, it might be wise to discuss our plans in more detail at some point, otherwise it may eventually result in two (or maybe even more) different addons each having their own branches of Rift research that are all similar but also slightly different, which may become confusing to some players.
- With directed enchantment, would that be akin to the Osmotic Enchanter from Thaumic Tinkerer?
